test/testcafe/framework/page-objects/DeviceChallengePollPageObject.js

```diff
@@ -42,6 +36,9 @@ export default class DeviceChallengePollViewPageObject extends BasePageObject {
   }
 
   getFooterSwitchAuthenticatorLink() {
+    if (userVariables.v3) {
+      return this.form.getLink('Verify with something else');
+    }
```

Comment:
I think you can use the function in the [BasePageObject](https://github.com/okta/okta-signin-widget/blob/0.1/test/testcafe/framework/page-objects/BasePageObject.js#L200-L202)